inhouse_bot/common_utils/is_admin.py

```diff
@@ -14,6 +14,9 @@ def admin_group_check():
     """
 
     def predicate(ctx: commands.Context):
+        if ctx.guild is None:
+            return False
+
```

Comment:
I think this case is impossible, this would be equivalently saying if the discord server didn't exist.

When do you think this case would ever be hit or rather how?

Reply:
I'm not exactly sure, but mypy was screaming at me. Here is the context definition:
![image](https://user-images.githubusercontent.com/13020251/208184261-15945a5d-1ea7-48ad-bc1d-5ee03811e3bd.png)